### ISSUE TITLE
fatal: repository 'https://cdn.cocoapods.org/' not found

### ISSUE THREAD

**Issue by anil1712** [andrehtissot/cordova-plugin-fcm-with-dependecy-updated#27]
I am trying to add this depecndency but getting this exception.

```
cordova plugin add cordova-plugin-fcm-with-dependecy-updated
Installing "cordova-plugin-fcm-with-dependecy-updated" for ios
Running command: pod install --verbose
Ignoring bcrypt-3.1.11 because its extensions are not built.  Try: gem pristine bcrypt --version 3.1.11

Ignoring byebug-9.0.6 because its extensions are not built.  Try: gem pristine byebug --version 9.0.6

Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3

Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3

Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3

Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3

Ignoring hitimes-1.2.4 because its extensions are not built.  Try: gem pristine hitimes --version 1.2.4

Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3

Ignoring nokogiri-1.6.8.1 because its extensions are not built.  Try: gem pristine nokogiri --version 1.6.8.1

Ignoring unf_ext-0.0.7.2 because its extensions are not built.  Try: gem pristine unf_ext --version 0.0.7.2

/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/executable.rb:89: warning: Insecure world writable dir /Users/apple/Library/Android/sdk/build-tools/28.0.2 in PATH, mode 040777

  Cloning into 'cocoapods-'...

  fatal: repository 'https://cdn.cocoapods.org/' not found

Failed to install 'cordova-plugin-fcm-with-dependecy-updated': Error: pod: Command failed with exit code 1 Error output:
Ignoring bcrypt-3.1.11 because its extensions are not built.  Try: gem pristine bcrypt --version 3.1.11
Ignoring byebug-9.0.6 because its extensions are not built.  Try: gem pristine byebug --version 9.0.6
Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3
Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3
Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3
Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3
Ignoring hitimes-1.2.4 because its extensions are not built.  Try: gem pristine hitimes --version 1.2.4
Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3
Ignoring nokogiri-1.6.8.1 because its extensions are not built.  Try: gem pristine nokogiri --version 1.6.8.1
Ignoring unf_ext-0.0.7.2 because its extensions are not built.  Try: gem pristine unf_ext --version 0.0.7.2
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/executable.rb:89: warning: Insecure world writable dir /Users/apple/Library/Android/sdk/build-tools/28.0.2 in PATH, mode 040777
  Cloning into 'cocoapods-'...
  fatal: repository 'https://cdn.cocoapods.org/' not found
    at ChildProcess.whenDone (/Users/apple/Projects/xymob/ionic/mts-apps/new-school-app/node_modules/cordova-common/src/superspawn.js:135:23)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:962:16)
    at Socket.stream.socket.on (internal/child_process.js:381:11)
    at Socket.emit (events.js:182:13)
    at Pipe._handle.close (net.js:606:12)
pod: Command failed with exit code 1 Error output:
Ignoring bcrypt-3.1.11 because its extensions are not built.  Try: gem pristine bcrypt --version 3.1.11
Ignoring byebug-9.0.6 because its extensions are not built.  Try: gem pristine byebug --version 9.0.6
Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3
Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3
Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3
Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3
Ignoring hitimes-1.2.4 because its extensions are not built.  Try: gem pristine hitimes --version 1.2.4
Ignoring json-1.8.3 because its extensions are not built.  Try: gem pristine json --version 1.8.3
Ignoring nokogiri-1.6.8.1 because its extensions are not built.  Try: gem pristine nokogiri --version 1.6.8.1
Ignoring unf_ext-0.0.7.2 because its extensions are not built.  Try: gem pristine unf_ext --version 0.0.7.2
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/executable.rb:89: warning: Insecure world writable dir /Users/apple/Library/Android/sdk/build-tools/28.0.2 in PATH, mode 040777
  Cloning into 'cocoapods-'...
  fatal: repository 'https://cdn.cocoapods.org/' not found
[ERROR] An error occurred while running subprocess cordova.
        
        cordova plugin add cordova-plugin-fcm-with-dependecy-updated exited with exit code 1.
        
        Re-running this command with the --verbose flag may provide more information.

```

**Comment by andrehtissot** [comment]
As I can see in the logs, thank you for sharing it, the cocoapods running is the `1.5.3`, which doesn't support the CDN source.

Please update the plugin with the new version.

**Comment by pajtai** [comment]
@andrehtissot Where do you update the cocoapods version?

**Comment by andrehtissot** [comment]
This plugin doesn't force a specific version of cocoapods to be installed. But it does change the source to use the CDN address if the installed cocoapods supports it.

This smart switch behavior was implemented in [v4.5.3](https://github.com/andrehtissot/cordova-plugin-fcm-with-dependecy-updated/releases/tag/v4.5.3). So, upgrading to that version would fix your problem.

Although, it is always recommended to keep up to newer versions, as bugs are fixed and features are added.
